packages/compass-global-writes/src/store/reducer.ts

```diff
@@ -224,62 +191,71 @@ export type RootState = {
   | {
       status: ShardingStatuses.LOADING_ERROR;
       shardKey?: ShardKey;
-      shardingError?: never;
-      pollingTimeout?: never;
       loadingError: string;
+      //////////////
+      userActionInProgress?: never;
+      shardingError?: never;
     }
   | {
       status: ShardingStatuses.NOT_READY;
+      //////////////
```

Comment:
I thought it might make it easier to read when we have the properties that aren't relevant for that state separated from those that are. But it's true that without description it's hard to pick up on that intent and it might get lost anyway 🙈